stan/math/prim/fun/lb_constrain.hpp

```diff
@@ -221,6 +221,29 @@ inline auto lb_constrain(const std::vector<T>& x, const std::vector<L>& lb,
   return ret;
 }
 
+/**
+ * Specialization of `lb_constrain` to apply a container of lower bounds
```

Comment:
Like I can leave the function undocumented and it will repeat the last one? I think with your other changes the docs for this function have changed a bit so I'd like to keep the docs that are here now.